core/src/main/java/io/questdb/cutlass/http/HttpResponseSink.java

```diff
@@ -696,33 +695,34 @@ public void clear() {
         public void sendStatusJsonContent(
                 int code
         ) throws PeerDisconnectedException, PeerIsSlowToReadException {
-            sendStatusJsonContent(code, null, null, null, null, true);
+            sendStatusJsonContent(code, null, null, null, null, -1L, true);
         }
 
         public void sendStatusJsonContent(
                 int code,
-                @Nullable CharSequence message
+                @Nullable Utf8Sequence message
         ) throws PeerDisconnectedException, PeerIsSlowToReadException {
-            sendStatusJsonContent(code, message, null, null, null, true);
+            sendStatusJsonContent(code, message, true);
         }
 
         public void sendStatusJsonContent(
                 int code,
-                @Nullable CharSequence message,
+                @Nullable Utf8Sequence message,
                 boolean appendEOL
         ) throws PeerDisconnectedException, PeerIsSlowToReadException {
-            sendStatusJsonContent(code, message, null, null, null, appendEOL);
+            sendStatusJsonContent(code, message, null, null, null, message != null ? message.size() : -1L, appendEOL);
```

Comment:
we should test for null message, `contentLength = -1` translates to "chunked" response, which is untrue in this context